wallet/src/main/java/io/neow3j/wallet/Nep5.java

```diff
@@ -0,0 +1,85 @@
+package io.neow3j.wallet;
+
+import io.neow3j.contract.ContractParameter;
+import io.neow3j.contract.ScriptHash;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.core.methods.response.NeoInvoke;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.List;
+
+public class Nep5 {
+    private Neow3j neow3j;
+    private ScriptHash fromContractScriptHash;
+
+    private Nep5(Builder builder) {
+        this.neow3j = builder.neow3j;
+        this.fromContractScriptHash = builder.fromContractScriptHash;
+    }
+
+
```

Comment:
You should use a single line space between methods. Please, change it for all the files you've committed.